pgxn/neon/walproposer.h

```diff
@@ -714,6 +714,9 @@ typedef struct WalProposerConfig
 	 */
 	char	   *safekeepers_list;
 
+	/* Extra libpq connection info options. */
+	const char *safekeeper_extra_conninfo_options;
```

Comment:
nit: I personally like that it's `const`, but since all other fields are not-const strings, maybe it makes sense to keep it consistent